deployment/docker/docker-compose.yml

```diff
@@ -70,7 +70,7 @@ services:
       - '--rule=/etc/alerts/*.yml'
       # display source of alerts in grafana
       - '-external.url=http://127.0.0.1:3000' #grafana outside container
-      - '--external.alert.source=explore?orgId=1&left=["now-1h","now","VictoriaMetrics",{"expr":"{{$$expr|quotesEscape|pathEscape}}"},{"mode":"Metrics"},{"ui":[true,true,true,"none"]}]' ## when copypaste the line be aware of '$$' for escaping in '$expr'
+      - '--external.alert.source=explore?orgId=1&left=[\"now-1h\",\"now\",\"VictoriaMetrics\",{\"expr\": \"{{$expr|quotesEscape|crlfEscape|queryEscape}}\"},{\"mode\":\"Metrics\"},{\"ui\":[true,true,true,\"none\"]}]'
```

Comment:
unsure about this change. When I try to run docker-compose I get the following error:
```
vmalert            | 2021-03-09T18:47:54.730Z   fatal   VictoriaMetrics/app/vmalert/main.go:79  failed to init: failed to init `external.alert.source`: error validating source template explore?orgId=1&left=[\"now-1h\",\"now\",\"VictoriaMetrics\",{\"expr\": \"{{|quotesEscape|crlfEscape|queryEscape}}\"},{\"mode\":\"Metrics\"},{\"ui\":[true,true,true,\"none\"]}]: errors(1): key "tpl", template "explore?orgId=1&left=[\\\"now-1h\\\",\\\"now\\\",\\\"VictoriaMetrics\\\",{\\\"expr\\\": \\\"{{|quotesEscape|crlfEscape|queryEscape}}\\\"},{\\\"mode\\\":\\\"Metrics\\\"},{\\\"ui\\\":[true,true,true,\\\"none\\\"]}]": error parsing annotation: template: :1: unexpected "|" in command
```

Reply:
```
docker-compose --version
docker-compose version 1.27.4, build 40524192
```